applications/pelion_client/src/events/Kconfig

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Event options"
+
+comment "Initially logged events"
+
+config PELION_INIT_LOG_NET_STATE_EVENT
+	bool "Log net state event"
+	depends on LOG
+	default y
+
+config PELION_INIT_LOG_PELION_STATE_EVENT
+	bool "Log Pelion state event"
+	depends on LOG
+	default y
+
+config PELION_INIT_LOG_PELION_CREATE_OBJECTS_EVENT
```

Comment:
`s/_EVENT/_EVENTS`
`s/state event"/state events"`
It's done similarly in the CAF, it would be consistent.

Reply:
why events? there is only one event?

Reply:
I see - we always use this form.

Reply:
Fixed